core/tests/test_extractor.py

```diff
@@ -5,7 +5,7 @@
 
 from core.extract.extractor import Extractor
 
-from core.util.audit_logs_parsing import Log
+from core.replay.connections_parser import Log
```

Comment:
Fix for using connectionLog from `connection_parser.py` and removed `audit_logs_parsing`